app/src/main/res/values/strings.xml

```diff
@@ -991,4 +991,27 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_drinking_water_potable_unsigned">Safe to drink (unsigned)</string>
     <string name="quest_drinking_water_not_potable_signed">Not safe to drink (signed)</string>
     <string name="quest_drinking_water_not_potable_unsigned">Not safe to drink (unsigned)</string>
+    <string name="quest_barrier_type_title">What type of barrier is this?</string>
+    <string name="quest_barrier_type_gate">Gate</string>
+    <string name="quest_barrier_type_lift_gate">Boom barrier</string>
+    <string name="quest_barrier_type_bollard">Bollard(s)</string>
+    <string name="quest_barrier_type_chain">Chain</string>
+    <string name="quest_barrier_type_rope">Rope</string>
+    <string name="quest_barrier_type_wire_gate">Removable wire section</string>
+    <string name="quest_barrier_type_cattle_grid">Grid blocking livestock from walking over</string>
+    <string name="quest_barrier_type_block">Big immobile object</string>
+    <string name="quest_barrier_jersey_barrier">Jersey barrier</string>
+    <string name="quest_barrier_type_log">Log</string>
+    <string name="quest_barrier_type_kerb">Curb</string>
+    <string name="quest_barrier_type_height_restrictor">Height barrier</string>
+    <string name="quest_barrier_full_height_turnstile">Full height turnstile</string>
+    <string name="quest_barrier_type_turnstile">Turnstile</string>
+    <string name="quest_barrier_type_passage">Passage through wall/fence/etc</string>
```

Comment:
@matkoniecz `etc.` should end with a period: https://en.wikipedia.org/wiki/Etc.

Reply:
Thanks, fixed in code in https://github.com/streetcomplete/StreetComplete/commit/f0621e6200179694fb3eb43ca62df96b18e877b6 and checked in POEditor.